autocertkit/network_tests.py

```diff
@@ -675,19 +675,34 @@ def _run_test(self, session, direction):
 
         log.debug("Client IPerf VM ref: %s" % client)
         log.debug("Server IPerf VM ref: %s" % server)
-            
-        log.debug("About to run iperf test...")
 
-        #Run an iperf test - if failure, an exception should be raised.
-        iperf_data = IperfTest(session, client, server, 
+        attempt_count = 0
+        fail_count = 0
+        max_failure_allowed = 1
```

Comment:
Hi can we make this configurable as in member variable or function parameter (with default value)?